ui/app/routes/vault/cluster.js

```diff
@@ -48,7 +48,8 @@ export default Route.extend(ModelBoundaryRoute, ClusterRoute, {
 
   getClusterId(params) {
     const { cluster_name } = params;
-    const cluster = this.modelFor('vault').findBy('name', cluster_name);
+    const records = this.store.peekAll('cluster');
+    const cluster = records.filter((record) => record.name === cluster_name)[0];
```

Comment:
I think i would instead use the JS-native `find` method, with your same logic here but then we can remove the `[0]`

They do the same thing but find is just a tad cleaner than filter 😁

Reply:
Good call! Thanks!